app/styles/courses/teacher-class-view.sass

```diff
@@ -0,0 +1,250 @@
+      height: 66px
+      
+    // Enroll student button
+    .enroll-student-button
+      margin-left: 33%
+      margin-left: calc((100% - 120px - 36px) * 0.6)
+
+  .inline-student-list
+    padding-left: 0
+    
+    // TODO: Fix highlighted text value
+    &:before
+      content: '('
+    &:after
+      content: ')'
+    list-style: none
+    
+    li
+      display: inline
+      a
+        white-space: nowrap
+    li:not(:last-child):after
+      content: ', '
+      
+  .student-info
+    max-width: 200px
+    line-height: 20px
+
+    .student-name, .student-email
+      overflow: hidden
+      text-overflow: ellipsis
+      white-space: nowrap
+    
+  .level-name
+    white-space: nowrap
+    overflow: ellipsis
+    font-weight: bold
+    // line-height: 20px
+  
+  .sort-button
+    border: none
+    background: none
+    font-weight: bold
+    text-decoration: underline
+
+  .edit-student-button
+    color: black
+    font-weight: bold
+    text-decoration: underline
+
+    .glyphicon
+      color: $grey-darker
+      
+  // Course Progress tab
+  
+  #course-progress-tab
+    .student-levels-table
+    .student-levels-row, .course-overview-row
+      padding-left: 75px
+      padding-top: 15px
+      padding-bottom: 30px
+    .student-levels-progress, .course-overview-progress
+      max-width: 880px
+      margin: -5px
+      margin-top: 5px
+    .progress-dot
+      margin: 5px
+  
+  // Checkboxes
+  // TODO: Move to style-flat?
+  
+  .checkbox-flat
+    position: relative
+    background: white
+    border: thin solid #979797
+    width: 20px
+    height: 20px
+    
+    input
+      visibility: hidden
+      
+    label
+      position: absolute
+      width: 18px
+      height: 18px
+      left: 1px
+      top: 1px
+
+    label:after
+      opacity: 0
+      content: ''
+      position: absolute
+      width: 14px
+      height: 7px
+      background: transparent
+      top: 3px
+      left: 1px
+      border: 2px solid black
+      border-top: none
+      border-right: none
+      transform: rotate(-45deg)
+
+    label:hover::after
+      opacity: 0.3
+    
+    input:checked + label:after
+      opacity: 1
+    
+  // stuff that feels specific to this page
+  .checkbox-flat
+    margin: 8px auto
+
+  // TODO: Move to style-flat?
+  .view-class-arrow
+    float: right
+    margin-right: 16px
+    color: $grey-darker
+    // height: 100%
+    font-size: 35px
+    line-height: 35px
+    align-self: center
+    .view-class-arrow-inner 
+      color: $grey-darker
+      &:hover
+        text-decoration: none
+    
+
+  // TODO: Scott is refactoring this
+    
+  .progress-dot
+    display: inline-block
+    margin-right: 6px
+    width: 34px
+    height: 34px
+    border-radius: 50%
+    // margin-top: 23px
+    // margin-bottom: 23px
+    background: $grey-dark
+    position: relative
+    .dot-label
+      padding-top: 2px
+      .dot-label-inner
+        font-size: 11px
+        font-weight: bold
+        color: white
```

Comment:
Hovers have been refactored to modified Bootstrap tooltips. This CSS is ready to be trimmed or reorganized.